src/pages/workspace/companyCards/WorkspaceCompanyCardsListHeaderButtons.tsx

```diff
@@ -39,12 +39,12 @@ function WorkspaceCompanyCardsListHeaderButtons({policyID, selectedFeed, shouldS
     const {translate} = useLocalize();
     const theme = useTheme();
     const {shouldUseNarrowLayout, isMediumScreenWidth} = useResponsiveLayout();
-    const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
+    const workspaceAccountID = getWorkspaceAccountID(policyID);
     const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
     const shouldChangeLayout = isMediumScreenWidth || shouldUseNarrowLayout;
-    const formattedFeedName = CardUtils.getCustomOrFormattedFeedName(selectedFeed, cardFeeds?.settings?.companyCardNicknames);
-    const isCustomFeed = CardUtils.isCustomFeed(selectedFeed);
-    const companyFeeds = CardUtils.getCompanyFeeds(cardFeeds);
+    const formattedFeedName = getCustomOrFormattedFeedName(selectedFeed, cardFeeds?.settings?.companyCardNicknames);
+    const isComercialFeed = isCustomFeed(selectedFeed);
```

Comment:
I am leaving most of the code to use the `Custom` terminology as the `Commercial` is more marketing word